moovingon_ai/manifest.json

```diff
@@ -0,0 +1,36 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "bb6dac8c-c4cd-414d-9b12-baef330a03a0",
+  "app_id": "moovingon-ai",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "moovingon.ai is a NOC orchestration and automation platform",
+    "title": "moovingon.ai",
+    "media": [
```

Comment:
3 images are required

Reply:
added

Reply:
Can you include an image of the out of the box dashboard? Thanks!

Reply:
addded
https://github.com/tomm24/integrations-extras/blob/add_moovingonai/moovingon_ai/images/moovingon_ai-overview-dashbard.png